libcipher/utils.py

```diff
@@ -0,0 +1,62 @@
+import io
+import sys
+import string
+
+
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is
+       English or not. Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file that the user provides
+    """
+
+    def __init__(self, dict_file):
+        self.dict_file = dict_file
+        self.LETTERS_AND_SPACE = string.ascii_letters + string.whitespace
+        content = ''.join(dict_file.readlines())
+        content = set(content.upper().split())   
+        self.ENGLISH_WORDS = content
```

Comment:
Not sure, why I would want to read them as lines and join the back! 3 lines above can be replaced with:

    lines = dict_file.readlines()
    self.english_words = set(word.upper().strip() for word in lines)